firefox_puppeteer/tests/test_toolbars.py

```diff
@@ -138,7 +137,10 @@ def test_matching_text(self):
             all_matches = title_matches + url_matches
             self.assertTrue(len(all_matches) > 0)
             for match_fragment in all_matches:
-                self.assertIn(match_fragment, (input_text, input_text.upper()))
+                # skip the new unified search item, if present
+                if match_fragment == '- Search with Yahoo':
+                    continue
```

Comment:
We cannot hard-code any search engine given that this will break localized builds of Firefox. Instead we should check for the special CSS class assigned to the result.